sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs

```diff
@@ -364,30 +362,40 @@ public override async Task<int> ReadAsync (byte [] buffer, int offset, int count
 						return 0;
 					}
 
-					using (var body = _status.Body) {
-						_reader = (JSObject)body.Invoke ("getReader");
+					try { 
+						using (var body = _status.Body) {
+							_reader = (JSObject)body.Invoke ("getReader");
+						}
+					} catch (JSException) {
+						cancellationToken.ThrowIfCancellationRequested ();
+						throw;
 					}
 				}
 
 				if (_bufferedBytes != null && _position < _bufferedBytes.Length) {
 					return ReadBuffered ();
 				}
 
-				var t = (Task<object>)_reader.Invoke ("read");
-				using (var read = (JSObject)await t) {
-					if ((bool)read.GetObjectProperty ("done")) {
-						_reader.Dispose ();
-						_reader = null;
+				try {
+					var t = (Task<object>)_reader.Invoke ("read");
+					using (var read = (JSObject)await t) {
+						if ((bool)read.GetObjectProperty ("done")) {
+							_reader.Dispose ();
+							_reader = null;
 
-						_status.Dispose ();
-						_status = null;
-						return 0;
-					}
+							_status.Dispose ();
+							_status = null;
+							return 0;
+						}
 
-					_position = 0;
-					// value for fetch streams is a Uint8Array
-					using (Uint8Array binValue = (Uint8Array)read.GetObjectProperty ("value"))
-						_bufferedBytes = binValue.ToArray ();
+						_position = 0;
+						// value for fetch streams is a Uint8Array
+						using (Uint8Array binValue = (Uint8Array)read.GetObjectProperty("value"))
+							_bufferedBytes = binValue.ToArray ();
+					}
+				} catch (JSException) {
+					cancellationToken.ThrowIfCancellationRequested ();
```

Comment:
When the request is aborted by disposing the response then don't throw `JSException` if the passed cancellation token is already cancelled.